pgmpy/tests/test_models/test_BayesianModel.py

```diff
@@ -226,6 +226,31 @@ def test_remove_nodes_from(self):
         self.assertRaises(ValueError, self.G1.get_cpds, 'diff')
         self.assertRaises(ValueError, self.G1.get_cpds, 'grade')
 
+    def test__eq__(self):
+        G4 = BayesianModel([('a', 'd'), ('b', 'd'),
+                                ('d', 'e'), ('b', 'c')])
+        cpd_a = TabularCPD('a', 2, [[0.2], [0.8]])
+        cpd_b = TabularCPD('b', 2, [[0.3, 0.7], [0.7, 0.3]],
+                       evidence=['a'],
+                       evidence_card=[2])
+        cpd_c = TabularCPD('c', 2, [[0.3, 0.7], [0.7, 0.3]],
+                       evidence=['b'],
+                       evidence_card=[2])
+        G4.add_cpds(cpd_a, cpd_b, cpd_c)
+        G5 = G4.copy()
```

Comment:
Rather than creating so many new networks you can simply use the networks created in the `setUp` method. For example you can simply use `self.G` and it's copy instead of `G4` and `G5`. Also you can modify those by adding or remove cpds/nodes/edges.  And since unittests runs the `setUp` method before running each test you don't need to worry about modifying the objects in setUp.